parquet-benchmarks/pom.xml

```diff
@@ -32,7 +32,7 @@
   <url>https://parquet.apache.org</url>
 
   <properties>
-    <jmh.version>1.3.4</jmh.version>
+    <jmh.version>1.9.3</jmh.version>
```

Comment:
Is there a reason that you updated to `1.9.3` instead of the latest `1.10` release?